test/columns.txt

```diff
@@ -1,2 +1,2 @@
-NAME	BUILDER	TYPE	STATUS	START	END
-.metadata.name	.status.builder	.status.conditions[*].state	.status.conditions[*].status	.status.startTime	.status.completionTime
+NAME	BUILDER	TYPE	STATUS	CREATED  START	END
```

Comment:
Is it possible to add columns to show startup duration and build execution duration? I don't know how powerful this template language is, if it's hard then don't worry about it, we can try to add it later.

Also, we can remove the `BUILDER` columns now that there's only one builder implementation.

Reply:
If the build has started but pod hasn't been scheduled then is it okay to have negative duration? Might be better off calculating that in status if we need that info.

Reply:
We should avoid a negative duration, since that would be pretty hard to interpret. I'm fine putting this off for some future change where we add startup and build duration to the `status` and make it easy to display here. Don't worry about it for now.